src/main/scala/com/danielasfregola/twitter4s/http/clients/streaming/statuses/TwitterStatusClient.scala

```diff
@@ -30,15 +31,19 @@ trait TwitterStatusClient extends TwitterStreamListenerHelper with StreamingOAut
     * @param locations : Empty by default. Specifies a set of bounding boxes to track.
     *                    For more information <a href="https://dev.twitter.com/streaming/overview/request-parameters#locations" target="_blank">
     *                      https://dev.twitter.com/streaming/overview/request-parameters#locations</a>
+    * @param language : Empty by default. A comma separated list of 'BCP 47' language identifiers.
+    *                    For more information <a href="https://dev.twitter.com/streaming/overview/request-parameters#language" target="_blank">
+    *                      https://dev.twitter.com/streaming/overview/request-parameters#language</a>
     * @param stall_warnings : Default to false. Specifies whether stall warnings (`WarningMessage`) should be delivered as part of the updates.
     * @param f: the function that defines how to process the received messages
     */
   def getStatusesFilter(follow: Seq[Long] = Seq.empty,
                         track: Seq[String] = Seq.empty,
                         locations: Seq[Double] = Seq.empty,
+                        language: Seq[Language] = Seq.empty,
```

Comment:
In other endpoints I have already sometimes slightly changed the name of the params to improve the usability of the client, so a little diversion would not be the end of the world. At the end our goal is having a client that is super-easy to use ;)

I think going for a little diversion of using the plural here would be acceptable....what do you think @lachatak?